docs/docs/develop/custom-apis/metrics-sql-api.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Metrics SQL API
+sidebar_label: Metrics SQL API
+sidebar_position: 30
```

Comment:
I think we can change `sidebar_position` to have the following sequence : `sql-api`, `metrics-sql-api` and `metrics-sql`.

Reply:
May be instead of keeping separate pages for `sql-api` and `metrics-sql-api` we can combine them in the main page itself. Just a thought.

Reply:
fixed positions, separate pages look more cleaner to me